Assignment/solution_student/index.html

```diff
@@ -8,24 +8,88 @@
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
+
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
 
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
-    
-      <header>
-        <h1>FEWD HTML Template</h1>
+
+      <header class="clearfix">
+        <div class="header-content">
```

Comment:
Great job using this inner container to achieve the centered-within-a-full-bleed-background! :+1: